.github/workflows/benchmark-tests.yml

```diff
@@ -21,6 +21,12 @@ jobs:
       - uses: actions/checkout@v2
```

Comment:
This action already checks out the `lingua-franca` repo (assuming that the workflow is invoked from the `lingua-franca` repo and not from elsewhere.

Reply:
you're right